src/test/groovy/org/mockito/release/notes/contributors/AllContributorsSerializerTest.groovy

```diff
@@ -0,0 +1,118 @@
+package org.mockito.release.notes.contributors
+
+import org.mockito.release.notes.model.ProjectContributor
+import spock.lang.Specification
+import spock.lang.Subject
+
+class AllContributorsSerializerTest extends Specification {
+
+    private File tempFile = new File(System.getProperty("java.io.tmpdir") + "/project-contributors.json")
+    @Subject serializer = new AllContributorsSerializer(tempFile)
+
+    def "serialization and deserialization of one contributor"() {
+        def contributors = new DefaultProjectContributorsSet()
+        def contributor = new DefaultProjectContributor("myName", "myLogin", "myProfileUrl", 5)
+        contributors.addContributor(contributor)
+
+        when:
+        serializer.serialize(contributors)
+        def actual = serializer.desrialize()
+
+        then:
+        actual.getAllContributors().containsAll(contributors.getAllContributors())
+        contributors.getAllContributors().containsAll(actual.getAllContributors())
+    }
+
+    def "serialization and deserialization of zero contributor"() {
+        def contributors = new DefaultProjectContributorsSet()
+
+        when:
+        serializer.serialize(contributors)
+        def actual = serializer.desrialize()
+
+        then:
+        actual.getAllContributors().containsAll(contributors.getAllContributors())
+        contributors.getAllContributors().containsAll(actual.getAllContributors())
+        actual.size() == 0
+    }
+
+    def "serialization and deserialization of two contributors"() {
+        def contributors = new DefaultProjectContributorsSet()
+        def contributor1 = new DefaultProjectContributor("myName 1", "myLogin 1", "myProfileUrl 1", 5)
+        def contributor2 = new DefaultProjectContributor("myName 2", "myLogin 2", "myProfileUrl 2", 5)
+        contributors.addContributor(contributor1)
+        contributors.addContributor(contributor2)
+
+        when:
+        serializer.serialize(contributors)
+        def actual = serializer.desrialize()
+
+        then:
+        actual.getAllContributors().containsAll(contributors.getAllContributors())
+        contributors.getAllContributors().containsAll(actual.getAllContributors())
+    }
+
+    def "serialization and deserialization of 5 contributors"() {
```

Comment:
Does testing against 5 sample contributors adds value to the coverage? I suspect that 2 sample contributors are enough to cover all complexity / code paths. I suggest to keep tests simple and test with as little sample data as possible to prove that the feature is working!